apps/ensnode/README.md

```diff
@@ -4,43 +4,18 @@
 
 ## Overview
 
-## goals
-
-> an optimized, multichain ens indexer that the community loves and integrates
-
-- ease of deployment for indiviudals to run their own infra
-- faster, more efficient, easier to use and deploy implementation
-- v1 — **high confidence in subgraph equivalency**
-  - 1:1 equivalency of results for queries via ensjs
-    - 100% ensjs, ens-app-v3 test suites passing
-    - should 'just work', following [this documentation](https://github.com/ensdomains/ensjs/blob/main/docs/basics/custom-subgraph-uris.md)
-  - ensjs equivalency confirmed via [ens-subgraph-transition-tools](https://github.com/namehash/ens-subgraph-transition-tools)
-- v2 — **optimized multichain indexer w/ unified namespace**
-  - true multichain indexing (mainnet, base, linea, etc)
-  - flattened, unified, multichain namespace
-  - support key ens-app-v3 and wallet ENS funtions via optimized resolvers & PRs
-  - high quality human-readable (healed) list of names by owner, necessary for many UX
-  - (possible) continued backwards compatibility with subgraph
-  - support indexing subset of data, i.e. only domains under parent node
-
-## notes
-
-- eth registry is ERC721, has many controllers (), no knowledge of pricing — delegated to registrar controllers
-- eth old registry & new registry migration due to security issue, new then fallback to old, therefore ignore all old evens on domains that have been seen by new registry
+ENSNode is a faster, more efficient, easier to use, and simpler to deploy ENS indexer. It can index mainnet, Basenames, and Linea, providing a unified multichain namespace in a subgraph-compatible GraphQL api. When indexing just mainnet, it has full data equivalency with the ENS Subgraph.
 
-### `eth` plugin performance
-
-estimated mainnet-only backfill time @ <=500rps = **~13 hours** on M1 Macbook (>10x speedup vs subgraph)
+- 1:1 equivalency of results for most queries
+  - 100% ensjs, ens-app-v3 test suites passing, via [ens-subgraph-transition-tools](https://github.com/namehash/ens-subgraph-transition-tools)
+  - [use ENSNode with ensjs](https://www.ensnode.io/guides/using-ensnode-with-ensjs/)
```

Comment:
Link is 404

Reply:
I now figured that this is not going to be 404 once we deploy the docs update to prod 👍